app/src/main/java/org/oppia/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -554,11 +583,12 @@ class StatePlayerRecyclerViewAssembler private constructor(
       hasConversationView,
       hasPrevious,
       previousNavigationButtonListener,
-      fragment as SubmitNavigationButtonListener
+      fragment as SubmitNavigationButtonListener,
+      shouldSplit
     )
     if (shouldSplit) {
       // "previous button" should appear in the conversation recycler view only
-      addPreviousButtonNavigation(hasPreviousButton, conversationPendingItemList)
+      addPreviousButtonNavigation(hasPreviousButton, conversationPendingItemList, shouldSplit)
```

Comment:
What is the difference between isSplitView and `shouldSplit`, because I think they are same, and atleast they should be kept same in 1 file.

Reply:
It is only the parameter name.  
The upcoming PR fixes this issue, it uses the observalbeField `isSplitView` instead of `shouldSplit`

Reply:
Okay, sounds good.